code/style.css

```diff
@@ -132,9 +148,20 @@ button {
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
+ 
 }
 
 button:hover {
-  opacity: 0.9;
-  transition: all 0.2s ease;
+    background-image: linear-gradient(
+      90deg,
+      #00c0ff 0%,
+      #ffcf00 49%,
+      #fc4f4f 80%,
+      #00c0ff 100%);
+      animation: slidebg 5s linear infinite;
+      
+}
+
+.thank-you {
+  background-color: black;
```

Comment:
I don't see any black bg on thankyou, think it's missing a closing curly bracket?